.github/workflows/sync-board.yaml

```diff
@@ -10,7 +10,9 @@ on:
 
 jobs:
   sync-board:
-    if: github.event.requested_reviewer.login == 'akeller' || github.event.requested_reviewer.login == 'christinaausley'
+    if: |
+      (github.event_name == 'pull_request_target' || !github.event.pull_request.head.repo.fork)
```

Comment:
🔧 If you remove the `on: pull_request` part of this workflow, you can also remove this line again

Reply:
Reverted in the latest commit!